Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:10
```

Comment:
is there a particular reason for node 10? i'd have a slight preference for 12, although either is fine. i believe node 12 does a better job sizing the heap by default, which is convenient since sourcecred can sometimes need a lot of heap.

Reply:
I think that I tried a newer version of node, and it had trouble building, but I can't be 100% certain it was because of the version. I chose node 10 based on looking at @Beanow Drone.io deployment in sfosc/sourcecred, so he might have comment about the choice of version.

Reply:
I'd be surprised if it had trouble with a new version of node--we run CI on both node 10 and 12, and I develop on node 12.

Reply:
I opted for 10 in the drone build for consistency. Node 12 is the current release and will ship new features. While node 10 is the active lts and only receives backports with additional scrutiny from the release group.

Both are within the test matrix of sourcecred so I opted for the more stable one. It should be no problem to make 12 work reliably though. As we only need to worry about testing the particular version we get with `FROM node:12` just before pushing a new release. Which is not the case for the drone build, because the node version and code we ship update independently.

Reply:
okay, if I'm hearing correctly, @decentralion has preference for 12 and @Beanow is okay and suggests testing - so I just tested building with 12 locally (works great!) and updated the PR. I'll put in a new PR later tomorrow (or worst case) this weekend with setup on CircleCI to deploy to sourcecred/sourcecred, and during that CI build @Beanow we will be sure to test before pushing a new release. Sound like a plan?

@Beanow I'm off to bed so if there are further tweaks needed, please go forth and do! Catch up tomorrow!

Reply:
We are good with the version so I'm going to resolve this particular conversation (it's getting a bit long on the threads!) Re-open if you see necessary.